common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -108,6 +108,24 @@ object Mappings {
       "filename" -> nonAnalyzedString
     )
 
+  val usageReference =
+    nonDynamicObj(
+      "referenceType" -> nonAnalyzedString,
+      "uri"  -> nonAnalyzedString,
+      "name" -> sStemmerAnalysedString
+    )
+
+  val usagesMapping =
+    nonDynamicObj(
+      "title"        -> sStemmerAnalysedString,
```

Comment:
Why analyse this field? I'd imagine we'd only ever search for exact article titles (phrase matching)?